webpack.config.js

```diff
@@ -9,6 +9,7 @@ var base = {
         host: '0.0.0.0',
         port: process.env.PORT || 8073
     },
+    devtool: 'source-map',
```

Comment:
> I haven't had too much of an issue with the current devtool, since at least in Chrome, I'm always able to figure out where the error is occurring.

Wow, really? For me it just doesn't do source mapping at all in Chrome without specifying `devtool`. That's not too bad since `playground.js` isn't minified, but still... I like source mapping. :)

Still, you have a point about `source-map` being the slowest option, and the fact that it requires external files complicates caching and ignoring. I'll experiment with the other options -- `cheap-module-eval-source-map` in particular -- and see how it goes.